crypto/fipsmodule/hkdf/hkdf_test.cc

```diff
@@ -416,3 +416,162 @@ TEST(HKDFTest, WycheproofSHA512) {
+  t->IgnoreAttribute("instring/10");
+  t->IgnoreAttribute("instring/11");
+  t->IgnoreAttribute("instring/12");
+  t->IgnoreAttribute("instring/13");
+  t->IgnoreAttribute("instring/14");
+  t->IgnoreAttribute("instring/15");
+  t->IgnoreAttribute("instring/16");
+  t->IgnoreAttribute("instring/17");
+  t->IgnoreAttribute("instring/18");
+  t->IgnoreAttribute("instring/19");
+  t->IgnoreAttribute("instring/20");
+  t->IgnoreAttribute("instring/21");
+  t->IgnoreAttribute("instring/22");
+  t->IgnoreAttribute("instring/23");
+  t->IgnoreAttribute("instring/24");
+  t->IgnoreAttribute("instring/25");
+  t->IgnoreAttribute("instring/26");
+  t->IgnoreAttribute("instring/27");
+  t->IgnoreAttribute("instring/28");
+  t->IgnoreAttribute("instring/29");
+  t->IgnoreAttribute("instring/30");
+  t->IgnoreAttribute("instring/31");
+  t->IgnoreAttribute("instring/32");
+  t->IgnoreAttribute("instring/33");
+  t->IgnoreAttribute("instring/34");
+  t->IgnoreAttribute("instring/35");
+  t->IgnoreAttribute("instring/36");
+  t->IgnoreAttribute("instring/37");
+  t->IgnoreAttribute("instring/38");
+  t->IgnoreAttribute("IterationVariableData");
+  t->IgnoreAttribute("IterationVariableData/2");
+  t->IgnoreAttribute("iv");
+  t->IgnoreAttribute("IVlen");
+  t->IgnoreAttribute("K(0)");
+  t->IgnoreAttribute("K(i)");
+  t->IgnoreAttribute("K(i)/2");
+  t->IgnoreAttribute("K(i)/3");
+  t->IgnoreAttribute("K(i)/4");
+  t->IgnoreAttribute("K(i)/5");
+  t->IgnoreAttribute("K(i)/6");
+  t->IgnoreAttribute("K(i)/7");
+  t->IgnoreAttribute("K(i)/8");
+  t->IgnoreAttribute("K(i)/9");
+  t->IgnoreAttribute("K(i)/10");
+  t->IgnoreAttribute("K(i)/11");
+  t->IgnoreAttribute("K(i)/12");
+  t->IgnoreAttribute("K(i)/13");
+  t->IgnoreAttribute("K(i)/14");
+  t->IgnoreAttribute("K(i)/15");
+  t->IgnoreAttribute("K(i)/16");
+  t->IgnoreAttribute("K(i)/17");
+  t->IgnoreAttribute("K(i)/18");
+  t->IgnoreAttribute("K(i)/19");
+  t->IgnoreAttribute("K(i)/20");
+  t->IgnoreAttribute("K(i)/21");
+  t->IgnoreAttribute("K(i)/22");
+  t->IgnoreAttribute("K(i)/23");
+  t->IgnoreAttribute("K(i)/24");
+  t->IgnoreAttribute("K(i)/25");
+  t->IgnoreAttribute("K(i)/26");
+  t->IgnoreAttribute("K(i)/27");
+  t->IgnoreAttribute("K(i)/28");
+  t->IgnoreAttribute("K(i)/29");
+  t->IgnoreAttribute("K(i)/30");
+  t->IgnoreAttribute("K(i)/31");
+  t->IgnoreAttribute("K(i)/32");
+  t->IgnoreAttribute("K(i)/33");
+  t->IgnoreAttribute("K(i)/34");
+  t->IgnoreAttribute("K(i)/35");
+  t->IgnoreAttribute("K(i)/36");
+  t->IgnoreAttribute("K(i)/37");
+  t->IgnoreAttribute("K(i)/38");
+
+  unsigned long l_len = std::stoul(l_str) / 8;
+
+  // HKDF_expand being used as a KDF in Feedback Mode
+  if (run_test) {
+    uint8_t *output = static_cast<uint8_t *>(new uint8_t[l_len]);
+
+    ASSERT_TRUE(HKDF_expand(output, l_len, md, ki.data(), ki.size(), fixed.data(), fixed.size()));
+    EXPECT_EQ(Bytes(ko.data(), ko.size()), Bytes(output, l_len));
+
+    delete[] output;
+    output = NULL;
```

Comment:
maybe remove this because it generates warnings "unused variable" warnings (output is assigned a value but never used, annoying but better to avoid the warnings...)